projects/media-viewer/src/lib/annotations/comment-set/comment-set.component.ts

```diff
@@ -196,4 +201,8 @@ export class CommentSetComponent implements OnInit, OnDestroy {
   clearSelection() {
     this.selectAnnotation = { annotationId: '', editable: false };
   }
+
+  allCommentsSaved() {
+    this.commentService.onCommentChange(this.commentComponents.some(comment => comment.editable === true));
```

Comment:
i like this logic however, there might be an issue with this (which isnt your change, just need to fix that bug). for example, i believe if youre editing and you click away, it closes the comment and it becomes false. tho it doesnt save, it will still mess with this logic.